docs/scripts/object_orientated_programming_poc/oop_poc_team.py

```diff
@@ -0,0 +1,57 @@
+"""The purpose of this script is to demonstrate how to use the new OOP interface for teams.
+The following actions are shown in this script:
+1. Creating a Team
+2. Instantiating a Team object from Synapse
+3. Getting information about the members of a Team
+4. Inviting a user to a Team
+5. Checking on invitations to join a Team
+6. Deleting a Team
+"""
+
+import asyncio
+
+from synapseclient.models.team import Team
+import synapseclient
+
+
+syn = synapseclient.Synapse(debug=True)
+syn.login()
+
+
+async def new_team():
+    # Create a team
+    new_team = Team(
+        name="python-client-test-team",
+        description="testing OOP interface",
+        can_public_join=False,
+    )
+    my_synapse_team = await new_team.create()
+    print(my_synapse_team)
+    # sleep for a bit to allow the team to be created (Synapse API slow?)
```

Comment:
I was unable to get this to run without a sleep here. It seemed like the next lines were running before the team was actually created.

Reply:
Weird... is 5 seconds arbitrary?  Would the same test would be to create a team and get the team right after without async?

Reply:
5 seconds was semi-arbitrary, 1 or 2 seconds did not work but 5 does.

Reply:
@BryanFauble @thomasyu888 This is the [response](https://sagebionetworks.jira.com/browse/PLFM-8240?focusedCommentId=196075) from platform about this delay with using `from_name` to retrieve a team. It appears as though this behavior is intended. Should I just document this in the docstring?

Reply:
Yes, that works for me! Let's document that it is the expected behavior for that method.